src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
```

Comment:
```suggestion
    private static final int MAX_BOUND = 100;
    private static final Random random = new Random();
    
    public Ball getRandomBall() {
        return new Ball(ColorSupplier.getRandomColor(), random.nextInt(MAX_BOUND));
```
You can use the constructor with all parameters and immediately return the newly created object

It's better to use constants instead of some numbers, makes code more understandable and easier to edit